docs/how-to/h-run-on-k8s-pod.md

```diff
@@ -1,73 +1,72 @@
-## Working with Charmed Kubernetes From within a pod
+## Working with Charmed Kubernetes from within a pod
 
 ### Setup
 
-After installing [Juju](https://juju.is/docs/olm/install-juju) and [Charmed Kubernetes](https://ubuntu.com/kubernetes/docs/install-manual) and [setting it up](https://ubuntu.com/kubernetes/docs/operations), let's discuss the steps to launch Spark jobs from within a pod in Charmed Kubernetes.
+After installing [Juju](https://juju.is/docs/olm/install-juju) and [Charmed Kubernetes](https://ubuntu.com/kubernetes/docs/install-manual) (together with applying [setup for the latter](https://ubuntu.com/kubernetes/docs/operations) ), now we can look into how to launch Spark jobs from within a pod in Charmed Kubernetes.
 
-First thing, we launch a pod using Canonical's Charmed Spark container image.
+First  we create a pod using Canonical's Charmed Spark container image.
 
-Create a pod manifest ```shell-demo.yaml``` for use with Charmed Kubernetes, something like this
+Edit the pod manifest file (we'll refer to it as ```shell-demo.yaml```) by adding the following conent:
 
 ```yaml
 apiVersion: v1
 kind: Pod
 metadata:
   name: shell-demo
-  namespace: spark-test-ns
+  namespace: default
 spec:
   containers:
   - name: spark-client
     image: ghcr.io/canonical/charmed-spark:3.4.0-22.04_edge
     command: ["/bin/pebble", "run", "--hold"]
-  serviceAccountName: spark-user
+  serviceAccountName: spark
   hostNetwork: true
   dnsPolicy: Default
 ```
 
-The pod can be created with
+The pod can be created with the following command
 ```shell
 kubectl apply -f shell-demo.yaml
 ```
 
-after that, one logs in with
+We can log into the pod as below
 
 ```shell
-$ kubectl exec --stdin --tty shell-demo -n spark-test-ns -- /bin/bash 
+$ kubectl exec --stdin --tty shell-demo -- /bin/bash 
 ```
 
-Now from within the pod, lets create the kubeconfig for use with spark-client. ```KUBECONFIG_CONTENTS_FROM_CHARMED_KUBERNETES``` will come from the first step executed outside the Charmed Kubernetes cluster pod. 
+Now let's create the Kubernetes configuration on the pod, with contents from the original server's `.kube/config` 
```

Comment:
is this step necessary? I think all the kubeconfig info is made available to the pod in downwards API

Reply:
Yes, I believe so. We have a ticket in the backlog, [DPE-1491](https://warthogs.atlassian.net/browse/DPE-1491), but this has not been addressed yet